apps/server/src/main.ts

```diff
@@ -20,6 +20,7 @@ const dirs = [
     'resources/coroutes',
     'resources/pdfs',
     'resources/images',
+    'resources/terrain',
```

Comment:
This will need to be its own folder for fragment to handle

Reply:
Outside of resources?

Reply:
Actually just get rid of it. That is for creating folders.

Reply:
done